analyze/analyses/paths/paths_emit.rs

```diff
@@ -1,113 +1,22 @@
 use std::io;
-use std::iter;
 
 use csv;
 
 use formats::json;
 use formats::table::{Align, Table};
 use twiggy_ir as ir;
-use twiggy_opt as opt;
 use twiggy_traits as traits;
 
-use super::paths_entry::PathsEntry;
-use super::Paths;
+use analyses::paths::Paths;
```

Comment:
Decided to be explicit about the path for this, so that it looks similar to the `use` statements within the helper module below. Totally subjective, happy to change this back if the `super::` syntax is preferable.